Update windows_installation_instructions.md (#129)
AlexsLemonade/reproducible-research: docs/setup_instructions/windows_installation_instructions.md

To install WSL, you will first need to use the Command Prompt.
Type `cmd` in the Windows search box to bring up the Command Prompt, then click the option in the right panel to "Run as administrator".

> (Note that all of the screenshots below were taken with Windows 10; if you have Windows 11, you may see some slight differences, but the steps should be the same)

Reviewer: ```suggestion
(Note that all of the screenshots below were taken with Windows 10; if you have Windows 11, you may see some slight differences, but the steps should be the same.)
```

Now you can use `Ctrl+Shift+C` and `Ctrl+Shift+V` to copy and paste in the Ubuntu window (just don't forget the `Shift`)!

> Bonus hint:

Reviewer: Lol wait was there no bonus hint here?

> Right clicking still pastes immediately though!

Reviewer: Not sure what "immediately" implies?

Author: clarifying?
```suggestion
Right clicking still pastes immediately (with no popup menu) though!
```

After installation is complete, we can install an extension to enable easier interaction between VS Code and WSL.
Open Visual Studio Code, and click the icon in the left side panel that looks like 4 squares (with one separated) to open the "Extensions" panel in the left panel.
In the search box that appears at the top of the left panel, type "WSL".
Select the "WSL" item from the list (with the penguin icon) and click the "Install" button.

> (In prior versions, this was called "Remotes - WSL", so the screenshot below is not quite accurate)

Reviewer: ```suggestion
(The screenshot below was taken with Windows 10 when this button was called "Remotes - WSL.")
```
